app/data/census_individual.json

```diff
@@ -160,11 +79,11 @@
                 },
                 {
                     "id": "date-of-birth",
-                    "title": "John Doe",
+                    "title": "{{ answers.person_full_name }}",
```

Comment:
I don't think we should be setting the block title to the person's name. I don't think we use the block title anywhere, but if we did it would probably be in something like the HTML title element, and we don't want people's names in there.